packages/video_player_videohole/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.3
+
+* Implement function of selecting video, audio and text traks.
```

Comment:
need change to:
* Implement function of selecting video, audio and text tracks.

Reply:
function → functionality?

Also consider increasing the minor version when adding new public APIs.

Reply:
fixed.
[Update CHANGELOG.md](https://github.com/flutter-tizen/plugins/pull/607/commits/e9fba847f4d972df7eed9e369b6107b37ff48130)